app/lib/.server/llm/model.ts

```diff
@@ -1,9 +1,77 @@
+// @ts-nocheck
+// Preventing TS checks with files presented in the video for a better presentation.
+import { getAPIKey, getAWSCredentials } from '~/lib/.server/llm/api-key';
 import { createAnthropic } from '@ai-sdk/anthropic';
+import { createOpenAI } from '@ai-sdk/openai';
+import { createAmazonBedrock } from '@ai-sdk/amazon-bedrock';
+import { ollama } from 'ollama-ai-provider';
+import { createOpenRouter } from "@openrouter/ai-sdk-provider";
 
-export function getAnthropicModel(apiKey: string) {
+export function getAnthropicModel(apiKey: string, model: string) {
   const anthropic = createAnthropic({
     apiKey,
   });
 
-  return anthropic('claude-3-5-sonnet-20240620');
+  return anthropic(model);
+}
+
```

Comment:
using openRoute you can use gemini but if you want to use gemini directly you need to add gemini provider
```js
export function getGeminiModel(apiKey: string, modelName: string = 'gemini-1.5-pro-latest') {
  const model = createGoogleGenerativeAI({
    apiKey,
  });

  return model(modelName);
}
```

Reply:
I responded with [e5d16df](https://github.com/stackblitz/bolt.new/pull/733/commits/e5d16dffd885ed9ba413c3423dff1cf4dbb53636)